inlong-dashboard/src/ui/components/FieldList/DetailModal.tsx

```diff
@@ -95,6 +95,65 @@ const Comp: React.FC<Props> = ({
     }),
   );
 
+  const sinkFieldList: ColumnsItemProps[] = [
+    {
+      title: i18n.t('meta.Sinks.SourceFieldName'),
+      dataIndex: 'sourceFieldName',
+      type: 'input',
+      rules: [
+        { required: true },
+        {
+          pattern: /^[a-zA-Z_][a-zA-Z0-9_]*$/,
+          message: i18n.t('meta.Stream.FieldNameRule'),
+        },
+      ],
+      props: (text, record) => ({
```

Comment:
If props does not need to rely on function parameters `text` and `record`, use them directly:
```
props: {
   disabled: true
}
```
This allows for better performance.